addons/marker_flags/XEH_postInit.sqf

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+
+if (!hasInterface) exitWith {};
+
+private _weapons = configProperties [configfile >> "CfgWeapons", QUOTE(isClass _x && (isText (_x >> QQGVAR(vehicle)))), true];
```

Comment:
This can take a while with large modsets, so we can do it at prestart once

```suggestion
private _weapons = (call (uiNamespace getVariable [QGVAR(flagItems), {[]}])) apply {configFile >> "CfgWeapons" >> _x};
```
and 
```
private _weapons = (configProperties [configfile >> "CfgWeapons", QUOTE(isClass _x && (isText (_x >> QQGVAR(vehicle)))), true]) apply {configName _x};
uiNamespace setVariable [QGVAR(flagItems), compileFinal str _weapons];
```